### PR TITLE
Showing % to building damage result

### LINKED ISSUE #261: Adding % (# of building) to table of building damage output
Adding % (# of building) to table of building damage output.
image

### PR DESCRIPTION
**Pull request by ylyangtw**
The changed page locates at: 3-4 => view results => building damage (only shows in finished runs) 

image
Added a switch on the top right corner to toggle the percentage.

### CODE REVIEW

File: src/components/childComponents/table/ResultTable.jsx [hunk 1]
```diff
@@ -88,6 +122,23 @@ export function ResultTable(props) {
...
+							onChange={(e) => handleChange("percent")}
+							color="default"
+							classes={{
+								switchBase: classes.switchBase,
+								checked: classes.checked,
+								track: classes.track
+							}}
+						/>
+					}
+					label="%"
```
Comment (longshuicy) [inline src/components/childComponents/table/ResultTable.jsx]: Could this be more descriptive? just the % label might confuse some folks

File: src/components/childComponents/table/ResultTable.jsx [hunk 2]
```diff
@@ -72,10 +84,32 @@ function calcClusterNumInCategory(fields) {
...
+		const collection = document.getElementsByClassName(classname);
+		for (let i = 0; i < collection.length; i++) {
+        	let x = collection[i];
+			if (x.style.visibility === "visible") {
+                x.style.visibility = "hidden";
+			} else {
+    			x.style.visibility = "visible";
+  			}
+    	}
+	}
```
Comment (longshuicy) [inline src/components/childComponents/table/ResultTable.jsx:103]: This works but it's not very "react" way. 
The React way would be have a state to set visible or not; then have two classes one has visibility set to none and the other not; 
When handle change, just simply set the state
And in the component you can do 

```
const [isVisible, setIsVisible] = useState(true);
...
isVisible ? 
<ToggleComponent className={classes.visibleClassName}> : <ToggleComponent className={classes.invisibleClassName}>
```

**Review by longshuicy — CHANGES_REQUESTED** [review]
The aesthetics looks great and functionality works too! 
Could we also default to % visible?
I also left some improvement suggestions. Thanks~

File: src/components/childComponents/table/ResultTable.jsx (hunk 2 above)
Comment (ylyangtw) [reply src/components/childComponents/table/ResultTable.jsx:103]: Thanks much Chen! I rewrote it in react style and the codes are simpler!

File: src/components/childComponents/table/ResultTable.jsx (hunk 1 above)
Comment (ylyangtw) [reply src/components/childComponents/table/ResultTable.jsx]: I changed % to 'View Percentage'. It's similar to 'View All' in the list of results

**Comment by ylyangtw** [comment]
Thanks for the suggestions! I have updated them~

**Review by longshuicy — APPROVED** [review]

**Pull request merged**